### DISCUSSION TITLE
Deadly Hunter fix in plans?

### DISCUSSION THREAD

**Discussion by aleeque** [diasurgical/DevilutionX#5400] [Ideas]
Definitely seems like a bug to me that the +200% dmg bonus is only applied to melee attacks, which the Rogue can't do when holding a bow.

Could this be introduced as an optional change maybe, so as to please both the purists and those who want to finally see the Rogue as a viable Hell/Hell character with similar clear speeds to the Warrior?

**Comment by StephenCWills** [comment]
It's already merged, actually. #3864

**Reply by aleeque** [reply]
Thanks for the reply!

So the +dmg affix was activated on this unique as a regular bug fix basically? If so, I'm kinda surpised this wasn't mentioned in any patch notes.

**Reply by AJenbo** [reply]
It's not part of any release yet.

**Reply by aleeque** [reply]
Oh I see.
